src/storekit.cs

```diff
@@ -955,4 +1004,141 @@ interface SKArcadeService {
 		[Export ("repairArcadeApp")]
 		void Repair ();
 	}
+
+
+	[Unavailable (PlatformName.MacCatalyst)]
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[Advice ("This API is not available when using UIKit on macOS.")]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface SKOverlayTransitionContext {
+		[Export ("addAnimationBlock:")]
+		void AddAnimationBlock (Action block);
+
+		[Export ("startFrame")]
+		CGRect StartFrame { get; }
+
+		[Export ("endFrame")]
+		CGRect EndFrame { get; }
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[DisableDefaultCtor]
+	[BaseType (typeof (NSObject))]
+	interface SKOverlayConfiguration {}
+
+	[Unavailable (PlatformName.MacCatalyst)]
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[Advice ("This API is not available when using UIKit on macOS.")]
+	[BaseType (typeof (SKOverlayConfiguration))]
+	[DisableDefaultCtor]
+	interface SKOverlayAppConfiguration {
+		[Export ("initWithAppIdentifier:position:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string appIdentifier, SKOverlayPosition position);
+
+		[Export ("appIdentifier", ArgumentSemantic.Retain)]
+		string AppIdentifier { get; set; }
+
+		[NullAllowed, Export ("campaignToken", ArgumentSemantic.Retain)]
+		string CampaignToken { get; set; }
+
+		[NullAllowed, Export ("providerToken", ArgumentSemantic.Retain)]
+		string ProviderToken { get; set; }
+
+		[Export ("position", ArgumentSemantic.Assign)]
+		SKOverlayPosition Position { get; set; }
+
+		[Export ("userDismissible")]
+		bool UserDismissible { get; set; }
+
+		[Export ("setAdditionalValue:forKey:")]
+		void SetAdditionalValue ([NullAllowed] NSObject value, string key);
+
+		[Export ("additionalValueForKey:")]
+		[return: NullAllowed]
+		NSObject GetAdditionalValue (string key);
+	}
+
+	[Unavailable (PlatformName.MacCatalyst)]
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[Advice ("This API is not available when using UIKit on macOS.")]
+	[BaseType (typeof (SKOverlayConfiguration))]
+	[DisableDefaultCtor]
+	interface SKOverlayAppClipConfiguration {
+		[Export ("initWithPosition:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (SKOverlayPosition position);
+
+		[NullAllowed, Export ("campaignToken", ArgumentSemantic.Retain)]
+		string CampaignToken { get; set; }
+
+		[NullAllowed, Export ("providerToken", ArgumentSemantic.Retain)]
+		string ProviderToken { get; set; }
+
+		[Export ("position", ArgumentSemantic.Assign)]
+		SKOverlayPosition Position { get; set; }
+
+		[Export ("setAdditionalValue:forKey:")]
+		void SetAdditionalValue ([NullAllowed] NSObject value, string key);
+
+		[Export ("additionalValueForKey:")]
+		[return: NullAllowed]
+		NSObject GetAdditionalValue (string key);
```

Comment:
I wonder if this should not be exposed as an indexer
`NSObject AdditionalValues [string key] { get; set; }`